src/System.Private.CoreLib/shared/System/Threading/WaitHandle.cs

```diff
@@ -82,18 +79,8 @@ public virtual IntPtr Handle
         [AllowNull]
         public SafeWaitHandle SafeWaitHandle
         {
-            get
-            {
-                if (_waitHandle == null)
-                {
-                    _waitHandle = new SafeWaitHandle(InvalidHandle, false);
-                }
-                return _waitHandle;
-            }
-            set
-            {
-                _waitHandle = value;
-            }
+            get => _waitHandle ??= new SafeWaitHandle(InvalidHandle, false);
```

Comment:
nice, I did not know that `??=` exists. However, I must say that it looks confusing a bit